7.2/Dockerfile

```diff
@@ -84,3 +84,11 @@ WORKDIR /usr/src/app
 RUN apt-get install -y libmagickwand-dev \
     && pecl install imagick \
     && docker-php-ext-enable imagick
+
+RUN apt-get install -y \
```

Comment:
can you please do this here: https://github.com/exozet/docker-php-fpm/pull/31/files#diff-56373fa7c1a1d5caa179088d26e72d29R47